lua/neogit/status.lua

```diff
@@ -573,26 +573,41 @@ end
 local stage = function()
   M.current_operation = "stage"
   local section, item = get_current_section_item()
+  local mode = vim.api.nvim_get_mode()
 
   if section == nil
     or (section.name ~= "unstaged" and section.name ~= "untracked" and section.name ~= "unmerged")
-    or item == nil then
+	or (mode.mode == "V" and item == nil) then
```

Comment:
Why did you add this condition?

Reply:
Because `stage_selection`, calls `generate_patch_from_selection` which expects `item` to not be `null`.